### PR TITLE
Pager styled with design tokens

### PR DESCRIPTION
**Pull request by mariannuar**
## Description
This PR styled the Pager component with design tokens

## How to review this pull request
- [ ] Run `npm run develop`
- [ ] Visit the [component](http://localhost:6006/?path=/story/components-pager--basic)
- [ ] Confirm it looks as expected in the [design](https://www.figma.com/file/TuQAuw81mBRsaepz2HP7BX/Emulsify-UI-Kit?type=design&node-id=23048-640&mode=dev)

Closes https://github.com/emulsify-ds/emulsify-ui-kit/issues/96

### CODE REVIEW

File: src/components/pager/pager.twig [hunk 1]
```diff
@@ -0,0 +1,103 @@
...
+ * - ellipses: If there are more pages than the quantity allows, then an
+ *   ellipsis before or after the listed pages may be present.
+ *   - previous: Present if the currently visible list of pages does not start
+ *     at the first page.
+ *   - next: Present if the visible list of pages ends before the last page.
+ *
+ * @see template_preprocess_pager()
+ */
+#}
+{% set pager__base_class = 'pager' %}
```
Comment (amazingrando) [inline src/components/pager/pager.twig:32]: What do you think about making this `pager__base_class = pager__base_class|default('pager')` to make it easy to override the base class?

File: src/components/pager/pager.twig [hunk 2]
```diff
@@ -0,0 +1,103 @@
...
+            <span {{ bem('visually-hidden') }}>{{ 'First page'|t }}</span>
+            <span aria-hidden="true">{{ items.first.text|default('First'|t) }}</span>
+          </a>
+        </li>
+      {% endif %}
+      {# Print previous item if we are not on the first page. #}
+      {% if items.previous %}
+        <li {{ bem('item', ['prev'], pager__base_class) }}>
+          <a {{ bem('link', ['prev'], pager__base_class) }} href="{{ items.previous.href }}" title="{{ 'Go to previous page'|t }}" rel="prev"{{ items.previous.attributes|without('href', 'title', 'rel') }}>
+            <span {{ bem('visually-hidden') }}>{{ 'Previous page'|t }}</span>
```
Comment (amazingrando) [inline src/components/pager/pager.twig:56]: If we are only adding a single class, should the span be `<span class="visually-hidden">` without the bem function? No need to change it. I'm just curious about the answer to the question.

**Review by amazingrando — APPROVED** [review]
@mariannuar I have a couple of questions that I am curious about. This code is approved. :shipit:

File: src/components/pager/pager.twig (hunk 2 above)
Comment (mariannuar) [reply src/components/pager/pager.twig:56]: @amazingrando Oh! I just copy-paste what it was already of the compound components. That's why it's like that, but you're right

**Pull request merged**